pkg/dns/server/name_table_test.go

```diff
@@ -104,6 +104,23 @@ func TestNameTable(t *testing.T) {
 		},
 	}
 
+	headlessServiceForServiceEntry := &model.Service{
```

Comment:
I do not understand how this service entry can produce ServiceInstance in real world.. should it have a workload selector?

Reply:
Label selectors were not specified - but for this test it does not matter because the service instances are added manually.